Interaction - Battlefields: Ghelsba Outpost, Ru'Hmet, Full Moon Fountain, Boneyard Gully

claywar opened the pull request:
<!-- Remove space and place 'x' mark between square [] brackets or click the checkbox after saving to affirm the following points: -->
<!-- (it should look like this: - [x] I have ...) -->
**_I affirm:_**
- [x] I understand that if I do not agree to the following points by completing the checkboxes my PR will be ignored.
- [x] I understand I should leave resolving conversations to the LandSandBoat team so that reviewers won't miss what was said.
- [x] I have read and understood the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/LandSandBoat/server/blob/base/CODE_OF_CONDUCT.md).
- [x] I have _**tested my code and the things my code has changed**_ since the last commit in the PR and will test after any later commits.

## What does this pull request do?
Converts and applies fixes to the following battlefields:
* Save the Children
* The Holy Crest
* Wings of Fury
* Petrifying Pair
* Toadal Recall
* When Angels Fall
* The Moonlit Path
* Moon Reading (Aijido now assists in battle)
* Head Wind
* Like the Wind
* Shell We Dance?
* A Sheep in Antlion's Clothing

<!-- Describe what your PR does here. If it closes an existing issue, you can mention: "Closes #1234" and GitHub will link this PR to that issue. -->

## Steps to test these changes
Enter/win/lose/exit all of the battlefields
<!-- Clear and detailed steps to test your changes here -->

claywar committed:
Interaction - Battlefield: Save the Children

Fixes Heir to the Light mission area reference

claywar committed:
Interaction - Battlefield: The Holy Crest

claywar committed:
Interaction - Battlefield: Wings of Fury

claywar committed:
Interaction - Battlefield: Petrifying Pair

claywar committed:
Interaction - Battlefield: Toadal Recall

claywar committed:
Interaction - Battlefield: When Angels Fall

claywar committed:
Interaction - Battlefield: The Moonlit Path

claywar committed:
Interaction - Battlefield: Moon Reading

claywar committed:
Interaction - Battlefield: Head Wind

claywar committed:
Interaction - Battlefield: Like the Wind

claywar committed:
Interaction - Battlefield: Shell We Dance?

Xaver-DaRed commented on sql/npc_list.sql:
```diff
@@ -27716,7 +27716,7 @@ INSERT INTO `npc_list` VALUES (17736072,'Tales_Beginning','Tales\' Beginning',11
 -- Bastok Markets (Zone 235)
 -- ------------------------------------------------------------
 
-INSERT INTO `npc_list` VALUES (17739777,'NPC[1]','NPC',171,-261.242,-12.136,80.643,0,50,50,0,0,0,6,27,0x0000620000000000000000000000000000000000,32,NULL,0);
```
This NPC is wrongly changed here. I PRd the fix a few days ago

claywar committed:
Interaction - Battlefield: Sheep in Antlion's Clothing

Remove unused variables

Fix Ghelsba Outpost Armoury Crate definitions

Add superlink to when angels fall

Fix Moon Reading phase 2 coordinates

Remove BST pet from required mobs in Head Wind

Fix armoury crates, invalid mob spawn function

Fix errant change
